support-data-rest/src/main/java/org/uniprot/api/literature/service/LiteratureService.java

```diff
@@ -68,6 +54,7 @@ public LiteratureEntry findById(final String literatureId) {
                 SolrRequest.builder()
                         .query("mapped_protein:" + accession)
                         .addSort(literatureSortClause.getSort(requestDTO.getSort(), false))
+                        .rows(BasicSearchService.DEFAULT_SOLR_BATCH_SIZE)
                         .build();
         QueryResult<LiteratureDocument> results =
                 repository.searchPage(solrRequest, requestDTO.getCursor(), requestDTO.getSize());
```

Comment:
So, here I got confused, because you have the rows attribute that is documented as "private int rows; // batch size of rows in solr request" and you are also passing it as parameter, so you are passing it twice to the repository code. Did you understand my point?

Reply:
removed the pagesize